packages/core/src/components/table/table-body-row/table-body-row.scss

```diff
@@ -60,3 +60,11 @@
     border-right: 1px solid var(--tds-table-divider);
   }
 }
+
+:host(.tds-table__row--clickable) {
+  cursor: pointer;
+}
+
+:host(.tds-table__row--clickable:focus-visible) {
+  outline: var(--focus-outline, 2px solid blue);
+}
```

Comment:
To answer your question about style of focus state:
Instead, you can use mixin from `packages/core/src/mixins/_focus-state.scss`.